### DISCUSSION TITLE
Add support for when AzureWebjobsStorage value is a keyvault secretUri for Linux Consumption

### DISCUSSION THREAD

**Discussion by georgekosmidis** [Azure/functions-action#140] [Feature asks]
The following error occurs when AzureWebjobsStorage value is a keyvault secretUri

Run Azure/functions-action@v1
  with:
    app-name: myFunctionApp
    package: ./app
    respect-pom-xml: false
    respect-funcignore: false
    scm-do-build-during-deployment: false
    enable-oryx-build: false
  env:
    AZURE_HTTP_USER_AGENT: 
    AZUREPS_HOST_ENVIRONMENT: 
Using RBAC for authentication, GitHub Action will perform resource validation.
Successfully acquired site configs from function app!
Detected function app sku: Consumption
Successfully acquired app settings from function app (RBAC)!
Detected function app language: Dotnet
Will archive ./app into /home/runner/work/_temp/temp_web_package_23554355612[15](https://github.com/.../runs/7393662757?check_suite_focus=true#step:4:15)72[18](https://github.com/.../runs/7393662757?check_suite_focus=true#step:4:18).zip as function app content
Will use WEBSITE_RUN_FROM_PACKAGE to deploy since RBAC is detected and your function app is on Linux Consumption.
**Error: Execution Exception (state: PublishContent) (step: Invocation)
Error:   At PublishContent, AzureWebjobsStorage : Failed to fetch AccountKey or AccountName.
Error: Deployment Failed!**

**Comment by patelchandni** [comment]
Please upvote this thread if you need this action to have support for when AzureWebjobsStorage value is a keyvault secretUri.

The current deployment is dependent on AzureWebJobsStorage for getting AccountKey and AccountName. There will be more investigation needed to access the storage account with keyvault secretUri. On adding support for using keyvault secretUri, users will need to use RBAC auth option for their workflow.
